pkg/controller/jobs/raycluster/raycluster_controller.go

```diff
@@ -87,6 +88,20 @@ func (j *RayCluster) GVK() schema.GroupVersionKind {
 	return gvk
 }
 
+func (j *RayCluster) IsDownsizable(wl *kueue.Workload) bool {
+	return j.GVK().Kind == "RayCluster" && features.Enabled(features.DynamicallySizedJobs) && JobHasDownsize(j.PodSets(), wl)
+}
+
+// JobHasDownsize checks whether a job has a scale down
+func JobHasDownsize(podSets []kueue.PodSet, wl *kueue.Workload) bool {
```

Comment:
```suggestion
func jobHasDownsize(podSets []kueue.PodSet, wl *kueue.Workload) bool {
```

Or is there a reason to export it?

Also, why not just implement inside `IsDownsizable`?